projects/openapi-cli/src/operations/patches/index.ts

```diff
@@ -48,7 +50,7 @@ export class OperationPatch {
     try {
       const result = JsonPatch.applyPatch(
         operation,
-        [...OperationPatch.operations(patch)],
+        JsonPatch.deepClone([...OperationPatch.operations(patch)]),
```

Comment:
This fixed all the shape diff issues we were having. Patches are mutated just like the spec is mutated. So simulating any patches that added new objects to the heap, was changing the patches that were actually applied on save. Nasty bug you hit if you have any non-trivial # of patches